src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -25,9 +25,11 @@
     font-family: "Comic Sans MS", sans-serif !important;
 }
 
-// UI Enhancer "Enlarge text" link options
+// UI Enhancer "Enlarge text" input/button options
 .fl-text-larger input,
-.fl-text-larger button {
+.fl-text-larger button,
+.fl-text-larger .fl-input-enhance,
+.fl-text-larger .fl-button-enhance, {
```

Comment:
I would lean towards: 
1. Use `fl-input-enhance` as it serves the same purpose; 
2. Don't support css classes that are not currently used to avoid them to slip out of our mind soon and sink into the css ocean. We can always add when they come into the real world.